README.md

```diff
@@ -75,7 +58,7 @@ Setup using Visual Studio Codespaces
     - choose view
     - choose terminal
 
-![alt text](./docs/images/codespaces4.jpg "Codespaces Menu Button")
+![Codespaces Menu Button](./docs/images/codespaces4.jpg "Codespaces Menu Button")
 
 > If using Codespaces, skip to the Azure login step
```

Comment:
If dotfiles are not enabled for github codespaces, should this step still be skipped? I suppose it'll be added soon but for the time being, the login will be needed.

Reply:
the step we're skipping is git clone - since you're in codespaces, the workspace is already setup

Reply:
ah can't read